src/xbps/index.md

```diff
@@ -105,3 +105,13 @@ To get a list of all installed packages, without their version:
 ```
 $ xbps-query -l | awk '{ print $2 }' | xargs -n1 xbps-uhelper getpkgname
 ```
+
+## Verifying RSA keys
+
+When installing Void for the first time, you may get a message from xbps claiming:
+
+```
+<REPO> repository has been RSA signed by <RSA-FINGERPRINT>
+```
+
+To verify the signature, ensure the <RSA-FINGERPRINT> matches one of the fingerprints in [void-packages](https://github.com/void-linux/void-packages/tree/master/common/repo-keys) or [void-mklive](https://github.com/void-linux/void-mklive/tree/master/keys).
```

Comment:
I'm pretty sure this line is beyond the maximum size.

Reply:
"matches one of the fingerprints in [...] and [...]" , because the keys in those repos should match.